### PR TITLE
Bump Travis go version to 1.9.1

### PR DESCRIPTION
**Pull request by ingvagabund**
SSIA

### CODE REVIEW

**Comment by aveshagarwal** [comment]
Can one of the admins verify this patch?

**Comment by ravisantoshgudimetla** [comment]
Thanks @ingvagabund but I think kube 1.9 is certified with go 1.9(https://github.com/kubernetes/community/blob/master/contributors/devel/development.md). Should we bump go to 1.10 before bumping kube?

**Comment by ingvagabund** [comment]
@ravisantoshgudimetla you have a point. Lemme update the PR.

File: hack/verify-gofmt.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ DESCHEDULER_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 GO_VERSION=($(go version))
 
-if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.2|go1.3|go1.4|go1.5|go1.6|go1.7|go1.8|go1.9') ]]; then
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.2|go1.3|go1.4|go1.5|go1.6|go1.7|go1.8|go1.9|go1.10') ]]; then
```
Comment (ingvagabund) [inline hack/verify-gofmt.sh]: Let's keep it here for the future bump to 1.10

**Pull request merged**